### PR TITLE
Improve 3d navigation with tootlip layers on mobile

### LINKED ISSUE #3011: touch sensitivity for tooltips in 3D
all iOS and Android touch devices
Add swisismage 
go 3d
zoom in and pan around

result
- the tooltip is activated event a very short touch 

-> extend the touch and hold timer

### PR DESCRIPTION
**Pull request by oterral**
Fix #3011 

[Test](https://mf-geoadmin3.dev.bgdi.ch/teo_touch/?)

### CODE REVIEW

**Comment by davidoesch** [comment]
What did u improve?

**Comment by oterral** [comment]
the tooltip doesn't appears everytime  you use the pinch

File: src/components/tooltip/TooltipDirective.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ goog.require('ga_topic_service');
   module.directive('gaTooltip',
       function($timeout, $http, $q, $translate, $sce, gaPopup, gaLayers,
           gaBrowserSniffer, gaDefinePropertiesForLayer, gaMapClick, gaDebounce,
-          gaPreviewFeatures, gaStyleFactory, gaMapUtils, gaTime, gaTopic) {
+          gaPreviewFeatures, gaStyleFactory, gaMapUtils, gaTime, gaTopic,
+          $window) {
```
Comment (gjn) [inline src/components/tooltip/TooltipDirective.js]: I can't see the need for `$window` here.

**Comment by gjn** [comment]
IMO, this is a sub-ideal workaround.

Can't we detect a pinch-start and differentiate it from a click? Maybe there is state info about this in the clicke event directly?

File: src/components/tooltip/TooltipDirective.js (hunk 1 above)
Comment (oterral) [inline src/components/tooltip/TooltipDirective.js]: No need

**Comment by oterral** [comment]
Needs more investigations on Cesium side for better fix.

**Pull request merged**